hack/aks/Makefile

```diff
@@ -257,6 +257,15 @@ dualstack-overlay-byocni-up: rg-up overlay-net-up ## Brings up an dualstack Over
 		--yes
 	@$(MAKE) set-kubeconf
 
+windows-nodepool-up: ## Add windows node pool
```

Comment:
Something I found while testing this, is for any scenario other than dualstack, we'll need to change the `--load-balancer-sku` to `standard`, if we want to add windows pools to it:
```
SUB=9bd7ff15-396a-4478-a89b-4d8ae7e302b6 AZCLI=az USER=paujohns make overlay-byocni-up
```
```
root[aks](master) SUB=9bd7ff15-396a-4478-a89b-4d8ae7e302b6 AZCLI=az USER=paujohns make windows-nodepool-up
az aks nodepool add -g paujohns-westus2 -n npwin \
        --node-count 2 \
        --node-vm-size Standard_B2s \
        --cluster-name  paujohns-westus2 \
        --os-type Windows \
        --subscription 9bd7ff15-396a-4478-a89b-4d8ae7e302b6
(SLBRequiredForMultipleAgentPools) Basic load balancers are not supported with multiple node pools. Create a cluster with standard load balancer selected to use multiple node pools, learn more at aka.ms/aks/nodepools.
Code: SLBRequiredForMultipleAgentPools
Message: Basic load balancers are not supported with multiple node pools. Create a cluster with standard load balancer selected to use multiple node pools, learn more at aka.ms/aks/nodepools.
make: *** [Makefile:261: windows-nodepool-up] Error 1
```

Reply:
So wherever you see `--load-balancer-sku basic` in this file, would become `--load-balancer-sku standard`

Reply:
very good tip John, thanks a lot!